Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CaesarCipher
+    {
+        public static void Demo()
+        {
+            bool isContinue = true;
+            while (isContinue)
+            {
+                int shift = UserSelectSizeAndDirectionOfShift();
+                string forOperation = RequestInput();
+                if (shift == 99)
+                {
+                    GenerateAllShiftIterations(forOperation);
+                }
+                else
+                {
+                    ConvertString(shift, forOperation);
+                }
+                isContinue = UserSelectsContinue();
+            }
+        }
+
+        private static void GenerateAllShiftIterations(string forOperation)
+        {
+            for (int i = 1; i <= 25; i++)
+            {
+                ConvertString(i, forOperation);
+            }
+        }
+
+        private static bool UserSelectsContinue()
+        {
+            Console.WriteLine("Please enter \"c\" to continue, any other key to exit");
+            string response = Console.ReadLine();
+            return ((response == "c") || (response == "C"));
```

Comment:
"c" and "C" are magic numbers. Please use a constant, so that it is easier to make changes. Also use a char instead of a string.